dev/io.openliberty.wsoc.2.2.internal_fat/test-applications/maxMessage.war/src/io/openliberty/wsoc/spec274/EchoServerEP.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.wsoc.spec274;
+
+import jakarta.websocket.OnMessage;
+
+import jakarta.websocket.server.ServerEndpoint;
+
+/*
+ * Echos messages sent to this endpoint.
+ * Used for testing MaxMessageTest
+ */
+@ServerEndpoint(value = "/echo")
+public class EchoServerEP {
```

Comment:
Since it doesn't echo anything -- can you rename it to MaxMessageSizeEP and change the url to "testMessageSize"

Or something along those lines?